### ISSUE TITLE
Use an external DNS service to automatically configure subdomains or custom domains

### ISSUE THREAD

**Issue by arschles** [kedacore/http-add-on#12]
### Use-Case

Currently, you get an external IP address for a new app you create. That means you have to manually configure DNS to complete the app creation process.

### Specification

It would be nice to remove that extra manual step. Possibly use https://github.com/kubernetes-sigs/external-dns to automate some/all of that process

**Comment by khaosdoctor** [comment]
Needs #36 and #10

**Comment by khaosdoctor** [comment]
I'll get on to it after #7

**Comment by arschles** [comment]
Out of scope. See #82
